README.md

```diff
@@ -18,7 +18,17 @@ Provide a stable API to call into.
 * OpenSSH 6.7+
 * Python dependencies in requirements.txt
 
-### Building
+### Install
+
+#### From pypi
+
+To install the latest version of python-podman published on pypi:
+
+```sh
+python3 -m pip install --user podman
```

Comment:
I would add an explanation that --user is the user's login. Most developer's wouldn't be logged into an account "podman". Or this wouldn't be needed it using virtualenv.

Reply:
Not sure to understand, the `--user` option just install the package in the current user namespace to avoid using `sudo` during install.
It's not an issue to use `--user` with virtualenv since the package will be installed in the virtualenv `sites-package` of the python interpreter that in use here.

Reply:
I was thinking we have a number of "users" in this ecosystem, so detailing which one to use in this context might help.  Vs. using the user for the service etc.